src/main/java/org/opensearch/security/auth/BackendRegistry.java

```diff
@@ -181,6 +180,8 @@ public void onDynamicConfigModelChanged(DynamicConfigModel dcm) {
         ipClientBlockRegistries = dcm.getIpClientBlockRegistries();
         authBackendClientBlockRegistries = dcm.getAuthBackendClientBlockRegistries();
 
+        SortedSet<AuthDomain> authDomains = Collections.unmodifiableSortedSet(dcm.getRestAuthDomains());
```

Comment:
qq, where is this field being used?

Reply:
This was leftover from rebasing after https://github.com/opensearch-project/security/pull/3975 was merged. I encountered the regression from that PR while investigating what it would take to implement this. TY for catching this, its now removed.